shared/firestore-models/util.ts

```diff
@@ -0,0 +1,27 @@
+import * as firebase from 'firebase-admin';
+
+import {getConfigRef, BranchManagerRepoConfig} from './repo-config'
+import {BranchManagerPullRequest} from './pull-request'
+
+
+export async function isTargetLabel(configId: string, label: string) {
```

Comment:
Superfluous line above, and maybe add a small comment about what this function does? 

From what does it check the label? etc. it's a bit unclear I think.